src/components/about/about.tsx

```diff
@@ -1,41 +1,46 @@
-import './about.css';
-
 export const About = () => (
-  <div className="page-wrapper">
-    <h2 className="about-title">About</h2>
-    <div className="about-text">
-      <p>
+  <div className="bg-[var(--primary-element-color)] rounded-[var(--container-border-radius)]">
+    <h2 className="pt-5 pr-5 pl-5 my-5 text-2xl text-[var(--primary-text-color)] font-bold md:pt-10 md:pr-10 md:pl-10">
+      About
+    </h2>
+    <div className="pr-5 pl-5 md:pr-10 md:pl-10">
+      <p className="my-4">
         About Welcome to our non-profit project, "Exploring the Solar System"! We are Dmytro Honcharov and Oleksii
         Moroz, two passionate software developers with a shared fascination for the vastness and beauty of our cosmic
         neighborhood.
       </p>
-      <p>
+      <p className="my-4">
```

Comment:
what a hack?! Don't you heard my class regarding styles for paragraphs??? Delete it!

Reply:
resolved